<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KWInsertPageDialog</class>
 <widget class="QWidget" name="KWInsertPageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>245</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Insert Page</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="beforeCurrent">
        <property name="text">
         <string>Before current page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="afterCurrent">
        <property name="text">
         <string>After current page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="afterLast">
        <property name="text">
         <string>After last page</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Page style:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pageStyle"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
